.github/workflows/release-embedding-sdk.yml

```diff
@@ -30,25 +30,21 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ inputs.git_ref }}
+          fetch-depth: 0
+          fetch-tags: true
```

Comment:
1. `fetch-tags: true` [doesn't work without](https://github.com/actions/checkout/issues/1471#issuecomment-1771231294) `fetch-depth: 0` and this looks like a bug.

2. I need this change because otherwise `git tag -l` wouldn't return anything because GitHub action does shallow fetch by default.